actors/verifreg/src/lib.rs

```diff
@@ -36,9 +42,12 @@ pub enum Method {
     AddVerifier = 2,
     RemoveVerifier = 3,
     AddVerifiedClient = 4,
-    UseBytes = 5,
-    RestoreBytes = 6,
+    UseBytes = 5,     // Deprecated
+    RestoreBytes = 6, // Deprecated
     RemoveVerifiedClientDataCap = 7,
+    RevokeExpiredAllocations = 8,
```

Comment:
The FIP has renamed this to `RemoveExpiredAllocations` to make conceptual room for other forms of revocation.